[
execute
.
py
]
[
test_abort_by_user_prompt_twice
[
alert
\
]
]
expected
:
FAIL
disabled
:
Bug
1459118
[
test_abort_by_user_prompt_twice
[
confirm
\
]
]
expected
:
FAIL
disabled
:
Bug
1459118
[
test_abort_by_user_prompt_twice
[
prompt
\
]
]
expected
:
FAIL
disabled
:
Bug
1459118
[
test_abort_by_user_prompt
[
confirm
\
]
]
expected
:
if
os
=
=
"
mac
"
:
[
"
PASS
"
"
FAIL
"
]
[
test_abort_by_user_prompt
[
alert
\
]
]
expected
:
if
os
=
=
"
mac
"
:
[
"
PASS
"
"
FAIL
"
]
[
test_abort_by_user_prompt
[
prompt
\
]
]
expected
:
if
os
=
=
"
mac
"
:
[
"
PASS
"
"
FAIL
"
]
[
test_abort_by_user_prompt_twice
[
confirm
\
]
]
expected
:
if
debug
and
(
os
=
=
"
win
"
)
:
[
"
PASS
"
"
FAIL
"
]
[
test_abort_by_user_prompt_twice
[
prompt
\
]
]
expected
:
if
(
processor
=
=
"
x86
"
)
and
debug
:
[
"
PASS
"
"
FAIL
"
]
[
test_abort_by_user_prompt_twice
[
alert
\
]
]
expected
:
if
(
processor
=
=
"
x86
"
)
and
debug
:
[
"
PASS
"
"
FAIL
"
]
